package.json

```diff
@@ -345,6 +345,7 @@
     "express": "^4.19.2",
     "fs-extra": "^11.1.1",
     "get-port": "^7.0.0",
+    "json-schema-faker": "^0.5.6",
```

Comment:
since new dependency added. Please check the CG to make sure there is no alert. and may be update thiredparty notice.

Reply:
No alert:
![1713762801590](https://github.com/microsoft/vscode-azureapicenter/assets/1050213/9ceb401f-40db-4f03-9a22-3391de9bb256)

And updated thiredparty notice.